### DISCUSSION TITLE
please add heltec balancer support

### DISCUSSION THREAD

**Discussion by oguzatagan** [fl4p/batmon-ha#154] [Q&A]
please add 24s heltec-neey active balancer support.

**Comment by fl4p** [comment]
can you post a supply link (aliexpress etc)?

**Reply by oguzatagan** [reply]
Tank you for your interest. 

Just found this amazing item on AliExpress. Check it out! 4.342,46TL  8%OFF | NEW VERSION 4TH NEEY 4A Smart Balancer 2~24S LFP Li-ion LTO Battery Pack Smart Active Balancer Lifepo4 Balancer
https://a.aliexpress.com/_mLUCLoC

https://github.com/NEEY-electronic/NEEY-electronic
